airflow/www/extensions/init_views.py

```diff
@@ -134,12 +134,29 @@ def init_error_handlers(app: Flask):
 
```

Comment:
https://github.com/apache/airflow/blob/master/airflow/www/extensions/init_views.py#L128-L133 is this dead code now?

Reply:
We override handler only for API, therefore in case we will build it w/o API default handler (with circles) should still be applied. 500 is handled properly by blueprint handler (with JSONify) btw, only 404 and 405 are not.